Library/Homebrew/extend/ENV/super.rb

```diff
@@ -190,7 +190,6 @@ def determine_dependencies
   def determine_cmake_prefix_path
     PATH.new(
       keg_only_deps.map(&:opt_prefix),
-      HOMEBREW_PREFIX.to_s,
```

Comment:
Potentially stupid question, but if we're removing this line shouldn't we be tweaking the line above to pass _all recursive deps_ via the `opt_prefix` instead of purely the `keg_only` ones?